### DISCUSSION TITLE
Estou tendo diversos problemas para implementar o Docker na minha máquina

### DISCUSSION THREAD

**Discussion by vitorafil** [balta-io/forum#562] [Tira Dúvidas] [answered]
Estava tendo o seguinte problema ao instalar o docker:
[WSL Update Failed](https://forums.docker.com/t/wsl-update-failed/140473)
Após seguir os passos do fórum listado, tive outro problema em minha máquina: 
![image](https://github.com/balta-io/forum/assets/143800250/a93ea466-fab9-4916-a4f0-f1a309e047fe)
O docker não é inicializado corretamente.

**Comment by andrebaltieri** [comment]
Bom dia, @vitorafil, como vai?
Você pode seguir sem Docker no Windows e usar o SQL Server Express:
https://www.microsoft.com/pt-br/sql-server/sql-server-downloads

**Reply by vitorafil** [reply]
Bom dia, Andre, pode me ajudar com a configuração inicial?
Estou com muita dificuldade pra configurar o ambiente do SQL, o Docker não funciona, tentei instalar o Ubuntu e não funciona, nada esta funcional no meu computador. Gostaria de seguir utilizando os mesmos padrões e ferramentas utilizados em aula.
Estou com esse problema pra instalar o Ubuntu:
![image](https://github.com/balta-io/forum/assets/143800250/ea50b5d8-a00f-4206-9e69-ea440e246ec3)
E mudando a região pra Estados Unidos não resolveu.

**Reply by andrebaltieri** [reply]
Bom dia, @vitorafil, como vai?

Não rola usar o SQL Express (Como comentei anteriormente)? Só usamos o Docker para o SQL!

**Reply by vitorafil** [reply]
Não encontrei como eu posso fazer esse processo de criação de usuário sem o docker
![image](https://github.com/balta-io/forum/assets/143800250/a092f6dd-ebb5-4e82-84fe-1def83289530)
Estou me baseando pelo artigo/video aula, não consigo utilizar o método de login SQL sem a criação do mesmo. Como posso fazer esse processo sem utilizar o docker?

**Reply by andrebaltieri** [reply, marked as answer]
Infelizmente não tenho um tutorial mostrando o processo sem o Docker.

Veja se este vídeo te ajuda:
https://www.youtube.com/watch?v=vimeN6uu8jU

**Reply by vitorafil** [reply]
Boa tarde Balta,
O vídeo ajudou, em certa parte do vídeo, você comenta sobre opções de virtualização, aí pesquisei e verifiquei na BIOS que minha opção referente à virtualização estava desabilitada:
![image](https://github.com/balta-io/forum/assets/143800250/eddcc6f8-dcbb-4973-9bd8-5764b5228345)
Após habilitar e reiniciar, meu docker funcionou normalmente, obrigado, e desculpe por perguntar tanto haha

**Reply by andrebaltieri** [reply]
Opaaaa, show demais @vitorafil, sensacional!
É isso, dá trabalho mesmo, mas parabéns por resolver 🚀
